includes/admin/class-wc-admin-post-types.php

```diff
@@ -405,7 +404,7 @@ private function quick_edit_save( $post_id, $product ) {
 		$manage_stock = ! empty( $_REQUEST['_manage_stock'] ) && 'grouped' !== $product->get_type() ? 'yes' : 'no'; // WPCS: input var ok, sanitization ok.
 		$backorders   = ! empty( $_REQUEST['_backorders'] ) ? wc_clean( $_REQUEST['_backorders'] ) : 'no'; // WPCS: input var ok, sanitization ok.
 		$stock_status = ! empty( $_REQUEST['_stock_status'] ) ? wc_clean( $_REQUEST['_stock_status'] ) : 'instock'; // WPCS: input var ok, sanitization ok.
-		$stock_amount = 'yes' === $manage_stock && ! empty( $_REQUEST['_stock'] ) ? wc_stock_amount( $_REQUEST['_stock'] ) : ''; // WPCS: input var ok, sanitization ok.
+		$stock_amount = 'yes' === $manage_stock && is_numeric( $_REQUEST['_stock'] ) ? wc_stock_amount( $_REQUEST['_stock'] ) : ''; // phpcs:ignore
```

Comment:
Still need to test with `isset` or `! empty`, that's why PHPCS it's still giving a warning, because we need always to check if exists before use, to avoid throwing errors.